include/grpc/grpc.h

```diff
@@ -387,7 +386,8 @@ grpc_call *grpc_channel_create_registered_call(
    completion of type 'tag' to the completion queue bound to the call.
    The order of ops specified in the batch has no significance.
    Only one operation of each type can be active at once in any given
-   batch. */
+   batch. 
+   THREAD SAFETY: grpc_call_start_batch is thread-compatible */
```

Comment:
There's a more complicated story here.

I think what we end up wanting is:
At most one thread can be starting a batch with GRPC_OP_SEND_* operations, and at most one thread can be starting a batch with GRPC_OP_RECV_* operations.